### ISSUE TITLE
Modify BTFS or run a lot of instances

### ISSUE THREAD

**Issue by hazcod** [rkfg/BTFS#1]
Hi,

first of all, great work!
I am working on making a dynamically generated directory listing backed by torrents, e.g.:
`/videos/Cat 1/Cat 1.mkv, /videos/Cat 2/Cat 2.mkv, /pictures/cats/cat.png` etc...

The data is pulled from a sqlite database where the magnet is stored as a string.

Would I be better off to modify your script so only one instance of btfs is running, or execute btfs for every  directory? (could go to the hundreds..)

TL;DR: Need a lot of instances of BTFS, might be better to modify it
